Teradata/covalent
# Feature/charts

## Description

PR for Bar & Line Charts

### What's included?

- Single Line Chart
- Multi Line Chart
- Bar Chart
- Responsive Charts
- Supports three formats: `json`, `csv`, `tsv`
- Default `Color Palette` for Bar Chart
- Drop Shadow in Bar Chart
- `Config Parameters` : Axis Titles, Chart Title, dataSrc for data source, contentType, zDepth, lineTitles & lineColors for Multi Line Chart

#### Test Steps

- Navigate to http://localhost:4200/#/components/bar-chart & http://localhost:4200/#/components/line-chart
- Check charts

### What's coming next
- [x] creating custom color palette using two colors provided by user
- [x] changing nomenclature for all the charts and bringing them under one td-chart element
- [x] data limit
- [x] line/bars shown on the page limit
- [x] more configs

##### Screenshots or link to CodePen/Plunker/JSfiddle
![covalent-charts](https://cloud.githubusercontent.com/assets/4469548/17881735/6734b29a-6925-11e6-94ed-08233574a148.gif)
